### ISSUE TITLE
Bug: disenchant value and distribution from items that can't be disenchanted

### ISSUE THREAD

**Issue by Bifidus** [shirsig/aux-addon#116]
eg:
lesser magic wand

**Comment by shirsig** [comment]
True. No way to fix this besides manually compiling a list of those items which is why I haven't done it so far (lots of work, possible unreliable across patch changes)

**Comment by mbloms** [comment]
Just had this problem with Greater Magic Wand too
